runtime/jvmti/jvmtiExtensionMechanism.c

```diff
@@ -1777,9 +1777,12 @@ jvmtiInternalGetStackTraceIteratorExtended(J9VMThread * currentThread, J9StackWa
 		
 		if (type & J9JVMTI_STACK_TRACE_EXTRA_FRAME_INFO) {
 			/* Fill in the extended data  */
+			int markInlined = type & J9JVMTI_STACK_TRACE_MARK_INLINED_FRAMES;
```

Comment:
Please never use `int`. There's really no need for the variable at all, as it's only used once.

Reply:
I've substituted the definition into the use site